SwissDataScienceCenter/renku-graph
# feat: made Xmx be configurable for Jena deployments

**Testing scenario:**
* fetch renku-graph from the `xmx-for-jena` branch;
* update your renku `requirements.yaml` file so graph repository points to the fetched renku-graph directory, e.g. `repository: "file://../../../renku-graph/helm-chart/renku-graph/"`;
* add desired `Xmx` value for jena into your `values.yaml` under `graph -> jena -> jvmXmx: 2G` (`2G` is the default);
* redeploy your environment using the modified `values.yaml`;
* wait for jena to re-deployed;
* ssh to the jena box and check the jena process args using e.g. `ps -ef`; expected is that the jena process should look like: `/usr/lib/jvm/java-1.8-openjdk/jre/bin/java -Xmx<desired value> -XX:+UseG1GC -jar /jena-fuseki/fuseki-server.jar`

closing #296
